core/corehttp/metrics.go

```diff
@@ -44,10 +43,6 @@ func MetricsOpenCensusCollectionOption() ServeOption {
 		view.RegisterExporter(pe)
 		view.SetReportingPeriod(2 * time.Second)
 
-		if err := view.Register(quicmetrics.DefaultViews...); err != nil {
```

Comment:
There is a failing sharness test because the metrics changed, could you update the test as well so we can see what the metrics now look like and so the tests will pass?

Reply:
This test is a mess. It doesn't work on OSX, due to subtle differences in `sed`.
And I can't get it running on Ubuntu. Let's see if a manually compiled list of metrics does the trick.

Reply:
Update: The problems on Linux were my fault ;)